lib/diagnostic.rb

```diff
@@ -31,31 +31,31 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = `#{character} is really a Sith Lord.`
```

Comment:
Ruby uses double quotes for string interpolation, the back ticks were what we used in JavaScript.